lib/3scale/backend/storage.rb

```diff
@@ -12,6 +12,12 @@ class Storage
       UnspecifiedURIScheme = Class.new Error
       private_constant :UnspecifiedURIScheme
 
+      # Constant used for when batching of operations
+      # is desired/needed. Batching is performed when a lot
+      # of storage operations are need to be performed and we
+      # want to minimize database blocking of other clients
+      BATCH_SIZE = 400
```

Comment:
We already have a constant related to this one in a different module: https://github.com/3scale/apisonator/blob/baa78aec5bb1c27f0e145ea20de0fa7893c5a5b5/lib/3scale/backend/constants.rb#L6

I think `BATCH_SIZE` should be where you placed it because it's clearly related to `Storage`. I think that's better than having a global constants module. But I also think that it would be clearer if we moved the other constant here. That could be done on a different PR.